.github/workflows/release_prep.yml

```diff
@@ -0,0 +1,82 @@
+name: "Release Prep"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Template version, must match the version for the PDK release it is being bundled with."
+        required: true
+        type: "string"
+
+jobs:
+  release_prep:
+    name: "Release prep"
+    runs-on: "ubuntu-20.04"
+
+    steps:
+
+      - name: "Checkout"
+        uses: "actions/checkout@v4"
+        with:
+          fetch-depth: 0
+
+      - name: "Generate changelog"
+        run: |
+          export GH_HOST=github.com
+          gh extension install chelnak/gh-changelog
+          gh changelog new --next-version v${{ github.event.inputs.version }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "setup ruby"
+        uses: "ruby/setup-ruby@v1"
+        with:
+          ruby-version: "3.2"
+          bundler-cache: "true"
+          bundler: 2.4.22
+
+      - name: "bundle environment"
+        run: |
+          echo ::group::bundler environment
+          bundle env
+          echo ::endgroup::
+
+      - name: "Update REFERENCE.md"
+        run: |
+          bundle exec rake strings:generate:reference
+
+      - name: "Get version"
+        id: "get_version"
+        run: |
+          echo "version=$(jq --raw-output .version metadata.json)" >> $GITHUB_OUTPUT
```

Comment:
this step is for the version of a module in metadata.json so can be removed too i'd say

Reply:
WOuldn't it also be passed to the changelog to set it there?

Reply:
it gets its version from the input -> line 27 of this workflow

Reply:
removed
Wonder why its retrieved 2 different ways?

Reply:
think that step retrieves the current version from a modules metadata.json so it can be updated later on in the module release workflow IIRC :) 

Anyways looks good - happy to merge